src/js/vic-v2/components/IntroductionPage.jsx

```diff
@@ -10,63 +10,143 @@ class IntroductionPage extends React.Component {
+                <ul>
+                  <li>Show a full front view of your face and neck, with full visibility of the face and no shadows, <strong>and</strong></li>
+                  <li>Be cropped from your shoulders up (much like a passport photo), <strong>and</strong></li>
+                  <li>Show you with your eyes open and a neutral expression, <strong>and</strong></li>
+                  <li>Be a square size and have a white or plain-color background (with no scenery or other people in the photo)</li>
+                  <li>Be uploaded as a .jpeg, .png, .bmp, or .tiff file</li>
+                </ul>
+              </div>
             </li>
-            <li className="process-step list-two">
+            {!signedIn && <li className="process-step list-two">
+              <div><h5>Sign In and Verify Your Identity</h5></div>
+              <p>Sign in to Vets.gov using your DS Logon or ID.me accounts. When you sign in to your account, we can verify your identity to make sure you’re you before we give you access to your personal information. This also helps to keep your information safe and prevent fraud and identity theft.</p>
+              <strong>If you’re signed in to your account, the application process can go more smoothly. Here’s why:</strong>
+              {accountBenefits}
+              <p>If you don’t have a DS Logon account, we’ll help you verify your identity using ID.me—our trusted technology partner that provides the strongest identity verification system available.</p>
+              {idProofingReqs}
+              <p><button className="usa-button usa-button-primary" onClick={() => this.props.saveInProgressActions.toggleLoginModal(true)}>Sign In or Create an Account</button></p>
+              <p><a href="/faq">Get more information about signing in to Vets.gov</a>.</p>
+              <div className="usa-alert usa-alert-info">
+                <div className="usa-alert-body">
+                  <h4 className="usa-alert-heading">Do I have to sign in to Vets.gov to apply for a Veteran ID Card?</h4>
+                  <div className="usa-alert-text">
+                    <p>No, but if you don’t sign in, it’ll take us longer to verify your identity and will delay a decision on your application. The fastest way to get your application processed is to sign in to your DS Logon or ID.me accounts.</p>
+                  </div>
+                </div>
+              </div>
+            </li>}
+            {signedIn && !idProofed && <li className="process-step list-two">
+              <div><h5>Verify Your Identity</h5></div>
+              <p>Verifying your identity is a security measure that helps us ensure that only you can access your Veteran ID Card. To do this, we use ID.me, our Veteran-owned trusted technology partner that provides the strongest identity verification system available. ID.me helps people like you easily prove your identity.</p>
```

Comment:
Should there be a comma between Veteran-owned and trusted?

Reply:
@cehsu Yes! thank you for catching this. It should be Veteran-owned, trusted....